### PR TITLE
Added Lekab extension

### PR DESCRIPTION
**Pull request by peekunnas**
Added lekab extension

### CODE REVIEW

File: extensions/lekab/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lekab",
+  "version": "2.1.0",
```
Comment (alexteusz) [inline extensions/lekab/package.json:3]: Extensions for Cognigy.AI start with version 4.0.0

File: extensions/lekab/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "lint": "tslint -c tslint.json src/**/*.ts"
+  },
+  "keywords": [
+    "Lekab",
+    "SMS"
+  ],
+  "author": "Petteri Paasikunnas / Lekab",
+  "license": "MIT",
+  "dependencies": {
+    "@cognigy/extension-tools": "^0.13.0",
```
Comment (alexteusz) [inline extensions/lekab/package.json:21]: Please use version 0.14.0

File: extensions/lekab/src/module.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { createExtension } from "@cognigy/extension-tools";
+
+import { sendSMSNode } from "./nodes/sendSMSNode";
+import { lekabConnection } from "./connections/lekabConnection";
+
+export default createExtension({
```
Comment (alexteusz) [inline extensions/lekab/src/module.ts:6]: Please provide the "options" property in order to give it a display name such as shown in this example: https://github.com/Cognigy/Extensions/blob/master/extensions/8x8/src/module.ts#L18

**Review by alexteusz — CHANGES_REQUESTED** [review]

File: extensions/lekab/package.json (hunk 1 above)
Comment (peekunnas) [reply extensions/lekab/package.json]: Thank you, version changed.

File: extensions/lekab/package.json (hunk 2 above)
Comment (peekunnas) [reply extensions/lekab/package.json]: Thanks, also this version is changed.

File: extensions/lekab/src/module.ts (hunk 3 above)
Comment (peekunnas) [reply extensions/lekab/src/module.ts:6]: options added.

File: extensions/lekab/package-lock.json
Comment (peekunnas) [inline extensions/lekab/package-lock.json:1]: changed versions (lekab & @cognigy/extensions-tools)

**Review by peekunnas — COMMENTED** [review]
Now all affected files should be changed according to your instructions.

**Review by alexteusz — APPROVED** [review]

**Pull request merged**